data/secscan_model/test/test_secscan_v4_model.py

```diff
@@ -15,37 +15,33 @@
 )
 from data.database import (
     Manifest,
-    Repository,
     ManifestSecurityStatus,
     IndexStatus,
     IndexerVersion,
-    User,
     ManifestBlob,
     db_transaction,
     MediaType,
 )
-from data.registry_model.datatypes import Manifest as ManifestDataType
 from data.registry_model import registry_model
 from util.secscan.v4.api import APIRequestFailure
-from util.canonicaljson import canonicalize
 from image.docker.schema2 import DOCKER_SCHEMA2_MANIFESTLIST_CONTENT_TYPE
 
 from test.fixtures import *
 
-from app import app, instance_keys, storage
+from app import app as application, instance_keys, storage
```

Comment:
`test.fixtures` provides a fixture named `app`, mypy gets confused when the `app` fixture is replaced with the `app` Flask object, it continues to think that `app` is a fixture.